analyzers/tests/SonarAnalyzer.UnitTest/Rules/RedundancyInConstructorDestructorDeclarationTest.cs

```diff
@@ -25,60 +25,67 @@ namespace SonarAnalyzer.UnitTest.Rules
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.CSharp10.Fixed.cs",
-                new RedundancyInConstructorDestructorDeclaration(),
-                RedundancyInConstructorDestructorDeclarationCodeFix.TitleRemoveConstructor,
-                ParseOptionsHelper.FromCSharp10);
+            codeFixBuilderRemoveConstructor.AddPaths("RedundancyInConstructorDestructorDeclaration.CSharp10.cs")
+                .WithCodeFixedPaths("RedundancyInConstructorDestructorDeclaration.CSharp10.Fixed.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp10)
+                .VerifyCodeFix();
+
 #endif
 
         [TestMethod]
         public void RedundancyInConstructorDestructorDeclaration_CodeFix_BaseCall() =>
-            OldVerifier.VerifyCodeFix<RedundancyInConstructorDestructorDeclarationCodeFix>(
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.cs",
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.BaseCall.Fixed.cs",
-                new RedundancyInConstructorDestructorDeclaration(),
-                RedundancyInConstructorDestructorDeclarationCodeFix.TitleRemoveBaseCall);
+            codeFixBuilderRemoveBaseCall.AddPaths("RedundancyInConstructorDestructorDeclaration.cs")
+                .WithCodeFixedPaths("RedundancyInConstructorDestructorDeclaration.BaseCall.Fixed.cs")
+                .VerifyCodeFix();
 
         [TestMethod]
         public void RedundancyInConstructorDestructorDeclaration_CodeFix_Constructor() =>
-            OldVerifier.VerifyCodeFix<RedundancyInConstructorDestructorDeclarationCodeFix>(
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.cs",
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.Constructor.Fixed.cs",
-                new RedundancyInConstructorDestructorDeclaration(),
-                RedundancyInConstructorDestructorDeclarationCodeFix.TitleRemoveConstructor);
+            codeFixBuilderRemoveConstructor.AddPaths("RedundancyInConstructorDestructorDeclaration.cs")
+                .WithCodeFixedPaths("RedundancyInConstructorDestructorDeclaration.Constructor.Fixed.cs")
+                .VerifyCodeFix();
 
         [TestMethod]
         public void RedundancyInConstructorDestructorDeclaration_CodeFix_Destructor() =>
-            OldVerifier.VerifyCodeFix<RedundancyInConstructorDestructorDeclarationCodeFix>(
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.cs",
-                @"TestCases\RedundancyInConstructorDestructorDeclaration.Destructor.Fixed.cs",
-                new RedundancyInConstructorDestructorDeclaration(),
-                RedundancyInConstructorDestructorDeclarationCodeFix.TitleRemoveDestructor);
+            new VerifierBuilder<RedundancyInConstructorDestructorDeclaration>()
+                .WithCodeFix<RedundancyInConstructorDestructorDeclarationCodeFix>()
+                .AddPaths("RedundancyInConstructorDestructorDeclaration.cs")
```

Comment:
Maybe we should place .AddPaths first?

Reply:
I would not do it here, as we are anyway not consistent and without a defined style this will not bring value.